site/gdocs/Image.tsx

```diff
@@ -83,6 +83,7 @@ export default function Image({
                 src={`/images/published/${d.filename}`}
                 alt={d.alt}
                 className={LIGHTBOX_IMAGE_CLASS}
+                loading="lazy"
```

Comment:
it feels a bit weird to put the attr on the `<img>` tag inside the `<picture>` tag, but [that's how it works](https://addyosmani.com/blog/lazy-loading/).